app/javascript/src/components/Dashboard/Notes/NewNoteForm.jsx

```diff
@@ -1,11 +1,14 @@
 import React from "react";
 
+import { Check } from "@bigbinary/neeto-icons";
 import { Formik, Form } from "formik";
-import { Button } from "neetoui";
-import { Input, Textarea } from "neetoui/formik";
-import * as yup from "yup";
+import { Input, Select } from "neetoui/formik";
+import { Button } from "neetoui/v2";
 
 import notesApi from "apis/notes";
+import FORM_INITIAL_VALUES from "constants/formInitialValues";
+import FORM_SELECT_OPTIONS from "constants/formSelectOptions";
+import FORM_VALIDATION_SCHEMAS from "constants/formValidationSchemas";
```

Comment:
Rather than importing it from an aliased constant file. Please create the same constants in a `constants.js` file in the scope of this directory. 
It's better to use aliased constants when we have constants that are being reused in multiple components.